samples/automl/automlTranslationModel.js

```diff
@@ -0,0 +1,481 @@
+          console.log('Translation model metadata:');
+          console.log(
+            '\tBase model: ',
+            model[i].translationModelMetadata.baseModel
+          );
+          console.log(
+            '\tSource language code: ',
+            model[i].translationModelMetadata.sourceLanguageCode
+          );
+          console.log(
+            '\tTarget language code: ',
+            model[i].translationModelMetadata.targetLanguageCode
+          );
+        } else if (
+          model[i].modelMetadata === 'textClassificationModelMetadata'
+        ) {
+          console.log(
+            'Text classification model metadata: ',
+            model[i].textClassificationModelMetadata
+          );
+        } else if (
+          model[i].modelMetadata === 'imageClassificationModelMetadata'
+        ) {
+          console.log('Image classification model metadata:');
+          console.log(
+            '\tBase model id: ',
+            model[i].imageClassificationModelMetadata.baseModelId
+          );
+          console.log(
+            '\tTrain budget: ',
+            model[i].imageClassificationModelMetadata.trainBudget
+          );
+          console.log(
+            '\tTrain cost: ',
+            model[i].imageClassificationModelMetadata.trainCost
+          );
+          console.log(
+            '\tStop reason: ',
+            model[i].imageClassificationModelMetadata.stopReason
+          );
+        }
+        console.log('Model create time:');
+        console.log('\tseconds: ', model[i].createTime.seconds);
+        console.log('\tnanos: ', model[i].createTime.nanos);
+        console.log('Model update time:');
+        console.log('\tseconds: ', model[i].updateTime.seconds);
+        console.log('\tnanos: ', model[i].updateTime.nanos);
+        console.log('Model deployment state: ', model[i].deploymentState);
+        console.log('\n');
+      }
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_translation_listModels]
+}
+
+function getModel(projectId, computeRegion, modelId) {
+  // [START automl_translation_getModel]
+  const automl = require('@google-cloud/automl');
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const modelId = `id of the model, e.g. “ICN12345”`;
+
+  // Get the full path of the model.
+  const modelFullId = client.modelPath(projectId, computeRegion, modelId);
+
+  // Get complete detail of the model.
+  client
+    .getModel({name: modelFullId})
+    .then(responses => {
+      const model = responses[0];
+
+      // Display the model information.
+      console.log('Model name: ', model.name);
+      console.log('Model id: ', model.name.split('/').pop(-1));

```

Comment:
This logic seems a bit complicated/arbitrary. Can it be either simplified or explained?